apps/explorer/src/components/validator-map/types.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+export interface Feature {
+    type: 'Feature';
+    id: string;
+    geometry: { coordinates: [number, number][][]; type: 'Polygon' };
+    properties: { name: string; alpha2: string };
```

Comment:
What does `alpha2` represent? Country code?

Reply:
yes, seems to be country code

Reply:
renamed it, doesn't seem to be used anywhere else